### ISSUE TITLE
Disable Debug Logging

### ISSUE THREAD

**Issue by DerTim1** [quantum-elixir/quantum-core#331]
In the new release Quantum logs to debug-level, e. g. 

`07:42:00.002 [debug] [:nonode@nohost][Elixir.Quantum.Executor] Execute started for job :check_services`

I think there is no possibility to have the global level set to debug and skip the messages of Quantum. So can you please add a configuration directive to disable the quantum debug-logging. Maybe something like that https://stackoverflow.com/questions/38196182/how-to-avoid-ecto-logs-inside-iex

Thanks for your advice.

**Comment by c-rack** [comment]
@DerTim1 thanks for reporting. Would you like to submit a PR?

**Comment by DerTim1** [comment]
Yes I can do this, but I would pull the cheap variant: a config option with debug_logging = true/false 

Defaultvalue? Maybe false to have the old behaviour?!

**Comment by maennchen** [comment]
@DerTim1 I think the default value should be enabled. In my opinion this is a good default value since it is on level debug.

**Comment by DerTim1** [comment]
@c-rack @maennchen 

I need some help: (1) Is the `util.ex` the right position to put a function `debug_logging` which only logs, if config-option is set to true? 
I already found the position to add the new config option. (2) But how can I access the `YourApp.Scheduler.config()` from the function in util.ex, becuase we don't know what _YourApp_ is set to.

**Comment by maennchen** [comment]
@DerTim1 You can put the function into the `util.ex` file.

There's no way to know what the config for a specific scheduler is without having the context. You'll need to pass the config option into the `init` function of the concerned stage.

**Comment by DerTim1** [comment]
@maennchen Sorry, but I think my knowledge about the structure of Quantum is too small to get this working.

**Comment by maennchen** [comment]
@DerTim1 It's not that complicated. Im going to drop you a few hints with whom you should be able to get this working:

 * [`Quantum.TaskStagesSupervisor. init /1`](https://github.com/quantum-elixir/quantum-core/blob/master/lib/quantum/task_stages_supervisor.ex) - This function starts the separate stages that are needed to schedule jobs
   - You can see that there are options passed to every entry in the list.
   - For example `Quantum.JobBroadcaster` is passed two config options.
 * The [`Quantum.JobBroadcaster.child_spec/1`](https://github.com/quantum-elixir/quantum-core/blob/master/lib/quantum/job_broadcaster.ex#L30) takes those arguments and creates a [worker spec](https://hexdocs.pm/elixir/Supervisor.html#module-child_spec-1).
 * The arguments are then passed into the [`Quantum.JobBroadcaster.start_link/1`](https://github.com/quantum-elixir/quantum-core/blob/master/lib/quantum/job_broadcaster.ex#L22)
 * The arguments are forwarded to the stages [`init/1`](https://github.com/quantum-elixir/quantum-core/blob/master/lib/quantum/job_broadcaster.ex#L35)
   - There you can put stuff into the stages state.

This process is more ore less the same for all stages. You can then get the config that was put into the state in every `handle_*`.

**Comment by maennchen** [comment]
@DerTim1 That was expected, Thank you!
